packages/design-system/src/components/Tooltip/Tooltip.jsx

```diff
@@ -310,7 +306,6 @@ Tooltip.propTypes = {
    * Sets the size of the invisible border around interactive tooltips that prevents it from immediately hiding when the cursor leaves the tooltip.
    */
   interactiveBorder: PropTypes.number,
-  inversed: PropTypes.bool,
```

Comment:
Also needs to be removed from Tooltip.d.ts file

Reply:
I'm not sure how to get that updated because `Tooltip.d.ts` gets generated when `yarn build` is run. 

There is still an inverse prop for TooltipIcon that needs to still be part of the component.

Reply:
If I delete `Tooltip.d.ts` file, it does not get re-generated when I run `yarn build` - Are you sure we can't edit directly, or is there something else to run to get them to re-generate?

Reply:
Oh I could be wrong @nichia can you weigh in here?

Reply:
Yes, typedef files are auto-generated for Typescript codes only (eg. InlineError, FormControl, Accordion). We manually add the typedef files for other React codes like `Tooltip` etc. 
However, since  we're keeping the `inversed` prop here for passing to `TooltipIcon`, looks like we're good here.